app/models/film_epk.rb

```diff
@@ -1,8 +1,8 @@
 class FilmEpk < ApplicationRecord
   belongs_to :user
-  has_many :awards
-  has_many :film_fams
-  has_many :presses
+  has_many :awards, dependent: :destroy
```

Comment:
So this looks like the delete will destroy all the associated records for a given EPK. Looks great